code_to_gast/helpers/js_helpers.py

```diff
@@ -73,11 +73,21 @@ def bool_op_to_gast(node):
 Used for functions called on objects and std funcs like console.log
 """
 def member_expression_to_gast(node):
-  if node.property.name == "log":
+  if node.object.name == "console" and node.property.name == "log":
```

Comment:
Can `console.log` follow a similar approach? If it's overly complex then I'd just create a taiga ticket to standardize this.

Reply:
Since console.log we structured slightly differently then the other standard function calls, it looks slightly different but I will add a taiga ticket